lib/player/player.go

```diff
@@ -140,6 +141,11 @@ func New(cfg *Config) (*Player, error) {
 		slog.With(teleport.ComponentKey, "player"),
 	)
 
+	ctx := context.Background()
+	if cfg.Context != nil {
+		ctx = cfg.Context
+	}
+
```

Comment:
Isn't it better to pass it as a first argument to `New`? I get `New` shouldn't get context, but it shouldn't start goroutines either. At least it'd be clearer it does something beyond creation. Or while changing the signature (adding a new argument) possibly renaming to `Start` or something is an option.

Reply:
I thought `Start(ctx context.Context, cfg *Config) (*Player, error)` could both create and start the Player (same as `New` does now). Would it be a big change to replace `New` with that?

Reply:
I agree splitting this would be ideal, but I think we don't share the same concern. My worry is that there is a chance someone will forget to pass the context again. But let's not drill this any longer.